packages/serverpod/lib/src/database/database_query.dart

```diff
@@ -349,3 +355,42 @@ _UsingQuery _usingQueryFromTableRelations(
     where: whereStatements.join(' AND '),
   );
 }
+
+void _validateTableReferences(
+  String tableName, {
+  List<Order>? orderBy,
+  Expression? where,
+}) {
+  List<String> exceptions = [];
```

Comment:
`exceptionMessages` would be a better name, as the list does not contain exceptions.